unattended_installer/install_functions/dashboard.sh

```diff
@@ -188,3 +188,18 @@ function dashboard_install() {
     fi
 
 }
+
+function dashboard_installDependencies() {
```

Comment:
If the dependencies cannot be installed we should show a warning saying that the report generation will fail

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/2122.